launch/jetson.launch

```diff
@@ -10,7 +10,7 @@
     <rosparam command="load" file="$(find mrover)/config/esw.yaml"/>
 
     <!-- IMU and GPS driver nodes -->
-    <node name="imu_driver" pkg="mrover" type="imu_driver.py" output="screen"/>
+    <!-- <node name="imu_driver" pkg="mrover" type="imu_driver.py" output="screen"/> -->
```

Comment:
Not sure if I missed this today, but what issues did this cause? Should it just be removed entirely?

Reply:
this is the problem where we need to run the IMU separately, I guess this also doesn't really fit in this PR but maybe we should just keep it until we can fix the IMU

Reply:
oh of course, sorry. Yeah I don't love that this is going in master but honestly it's probably better to have it there, and then we know we need to run imu_driver.py separately since imu doesn't work, rather than forget to make this edit locally and suffer the consequences